app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
 
-The examples below will use the most up-to-date version of Postgres, Postgres 11. To use 9.6 or 10, append "96" or "10" to every `postgres` command. 
+- Either the Kong RPM or Kong Yum repo file.
+- The license file `license.json`
 
-1. Install PostgreSQL
-  ```
-  $ sudo amazon-linux-extras install postgresql9.6
-  ```
-  or
-  ```
-  $ sudo amazon-linux-extras install posgresql10
-  ```
-  or
-  ```
-  $ sudo amazon-linux-extras install postgresql11
-  ```
+## Step 2. Install Kong Enterprise
 
-2. Install the PostgreSQL Server
-  ```
-  $ sudo yum install postgresql postgresql-server
-  ```
+### Option 1: If installing via a downloaded RPM package
+ 
+Execute a command similar to the following, using the appropriate RPM file name you downloaded.
 
-3. Enable the PostgreSQL service
-  ```
-  $ sudo systemctl enable postgresql.service
-  ```
+```bash
+$ sudo yum install kong-enterprise-edition-1.3.0.1.aws.rpm
+```
 
-4. Initialize the database
-  ```
-  $ sudo /usr/bin/postgresql-setup --initdb
-  ```
+### Option 2: If installing via the Yum repository
+   
+1. Move the repo file in your home directory to the /etc/yum.repos.d/ directory.
 
-5. Start PostgreSQL
-  ```
-  $ sudo systemctl start postgresql
-  ```
+    ```bash
+    $ sudo mv bintray--kong-kong-enterprise-edition-aws.repo /etc/yum.repos.d/
+    ```
 
-**ii. Create a Kong user and database**
+2. Begin the installation via the Yum repository:
 
-The user and database can easily be created via the command line by using PostgreSQL’s command line tool `psql`
+    ```bash
+    $ sudo yum update -y
+    $ sudo yum install kong-enterprise-edition -y
+    ```    
 
-1. Change to the `postgres` user
-  ```
-  $ sudo -i -u postgres
-  ```
+### Copy the License File
+ 
+Copy the license file from your home directory to the `/etc/kong` directory like so:
 
-2. Launch the PostgreSQL command line tool `psql`
-  ```
-  $ psql
-  ```
+```bash
+$ sudo cp license.json /etc/kong/license.json
+```
 
-3. Create the `kong` user and database
-  ```
-  $ psql> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
-  ```
-  > ⚠️Note: Make sure the username and password for the Kong Database are kept safe. We have used a simple example for illustration purposes only.
+## Step 3. Setup PostgreSQL
 
-4. Exit `psql`
-  ```
-  $ psql> \q
-  ```
+1. Install PostgreSQL.
 
-5. Return to the terminal
-  ```
-  $ exit
-  ```
-
-**iii. Connect Kong to the PostgreSQL database**
-
-Postgres uses `ident` authentication by default, to allow the `kong` user to communicate with the database locally we must change the authentication method to `md5` by modifying the Postgres configuration file. 
+    Follow the instructions avaialble at: https://www.postgresql.org/download/linux/redhat/ to install a supported version of PostgreSQL. Kong supports version 9.5 and higher. As an example, you may run a command set similar to:
```

Comment:
Change "Follow the instructions avaialble at: https://www.postgresql.org/download/linux/redhat/ to install a supported version of PostgreSQL. Kong supports version 9.5 and higher. As an example, you may run a command set similar to:" 
to 
"Follow the instructions available at https://www.postgresql.org/download/linux/redhat/ to install a supported version of PostgreSQL. Kong supports version 9.5 and higher. As an example, you can run a command set similar to: